packages/pinia/__tests__/mapHelpers.spec.ts

```diff
@@ -245,5 +245,34 @@ describe('Map Helpers', () => {
         'replaced replaced'
       )
     })
+
+    it('setup store', async () => {
+      const useSetupStore = defineStore('setup', () => {
+        const a = ref(true)
```

Comment:
Let's just have one single variable named `text` that starts as `'initial'` and the computed is `textUpper` which returns an uppercase value.